apps/frontend/src/hooks/useWalletConnect.ts

```diff
@@ -20,6 +20,12 @@ export const useWalletConnect = () => {
     await onboard.disconnectWallet();
   }, []);
 
+  const switchNetwork = useCallback(async (chainId: string | number) => {
```

Comment:
```suggestion
  const switchNetwork = useCallback(async (chainId: ChainId) => {
```

Note it's `ChainId` (which is union between ChainId and string) and not `ChainIds` (which is enum)